cluster/gce/config-test.sh

```diff
@@ -246,8 +246,22 @@ if [ ${ENABLE_IP_ALIASES} = true ]; then
   PROVIDER_VARS="${PROVIDER_VARS} ENABLE_IP_ALIASES"
 fi
 
+# Admission Controllers to invoke prior to persisting objects in cluster
+# If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
+ADMISSION_CONTROL="${KUBE_ADMISSION_CONTROL:-NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds}"
+
+# Optional: use a default, fairly open PodSecurityPolicy
+ENABLE_POD_SECURITY_POLICY="${ENABLE_POD_SECURITY_POLICY:-true}"
+
+if [[ "${ENABLE_POD_SECURITY_POLICY:-}" == "true" ]]; then
+  ADMISSION_CONTROL="${ADMISSION_CONTROL},PodSecurityPolicy"
```

Comment:
Before we start making individual options for admission plugins, let's consider how we want these to be used. How can we enable the following scenarios:
- completely specify the admission chain
- enable a plugin that is not on by default without worrying about order or having to fully specify the chain
- disable a plugin that is on by default without worrying about order or having to fully specify the chain

@deads2k @derekwaynecarr